libact/query_strategies/tests/test_realdata.py

```diff
@@ -153,7 +153,7 @@ def test_DensityWeightedUncertaintySampling(self):
         qs = DWUS(trn_ds, random_state=1126)
         qseq = run_qs(trn_ds, qs, self.y, self.quota)
         assert_array_equal(
-            qseq, np.array([30, 179, 104, 186, 28, 65, 142, 62, 257, 221]))
+            qseq, np.array([ 30, 179, 104, 186,  28,  65, 142,  62, 257, 221]))
```

Comment:
nit: Can we change the spacing style back to the original version?

Reply:
Sure! I have changed the spacing style back.